controllers/depresolver/depresolver_config.go

```diff
@@ -51,6 +52,7 @@ const (
 	LogFormatKey                   = "LOG_FORMAT"
 	LogNoColorKey                  = "NO_COLOR"
 	SplitBrainCheckKey             = "SPLIT_BRAIN_CHECK"
+	MetricsAddressKey              = "METRICS_ADDRESS"
```

Comment:
@kuritka this env variable should be wired in `operator.yaml` helm template too. Or is k8gb helm chart out of the scope of this change?

Reply:
good point  @somaritane 👍 , amended